packages/circuit-ui/components/ImageInput/ImageInput.tsx

```diff
@@ -51,7 +53,7 @@ export interface ImageInputProps
   /**
    * A callback function to call when the input is cleared.
    */
-  onClear: () => void;
+  onClear: (event: MouseEvent | KeyboardEvent) => void;
```

Comment:
Quoting the PR description 😝

> @justman00 and I are trying to use the ImageInput inside the Popover component. When clicking the button to clear the image, the popover is closed. We confirmed that the popover is closed by the `useOutsideClick` hook. We suspect this happens because the clear button is removed from the DOM before the outside click handler is executed, so it doesn't know that the clear button was inside the popover. In order to prevent this, we plan to use `event.stopPropagation()` in the `onClear` callback.

Reply:
🙈